plejd/ble.bluez.js

```diff
@@ -146,10 +147,7 @@ class PlejdService extends EventEmitter {
       console.log('plejd-ble: error: failed to start discovery. Make sure no other add-on is currently scanning.');
       return;
     }
-
-    setTimeout(async () => {
-      await this._internalInit();
-    }, this.connectionTimeout * 1000);
+    return new Promise(resolve => setTimeout(() => resolve(this._internalInit()), this.connectionTimeout * 1000));
```

Comment:
This allows us to actually wait for the `_internalInit` in `write` errorhandler so we do not retry into void.